### ISSUE TITLE
Truncated Log Display in Unraid

### ISSUE THREAD

**Issue by Tymarand** [guydavis/machinaris#267]
**Describe the bug**
When looking at the log from within Unraid Docker section, the first ~10 characters are cut off.

**To Reproduce**
Unraid > Docker tab > left click on Machinaris icon > Logs > missing log text

Steps to reproduce the behavior, e.g.:
1. Navigate to Unraid webUI
2. Click on Docker tab (assuming Machinaris is already installed)
3. Click on Machinaris icon to show options
4. Select Logs
5. See error

**Expected behavior**
It is expected to see all of the desired text without being cut off.

**System setup:**
 - OS: Unraid
 - Docker version number: 20.10.5
 - Machinaris branch: main
 - Machinaris version number: Machinaris 0.5.4

**Config**
Name:
machinaris
Overview:
Building upon the official Chia binaries, Machinaris also offers a plot manager, log monitor, and a simple WebUI for Unraid.

Repository:
ghcr.io/guydavis/machinaris
Network Type:

Custom : br0
Fixed IP address (optional):
192.168.1.251
Subnet: 192.168.1.0/24
Console shell command:

Shell
Privileged:
OFF
appdata:
/mnt/user/appdata/machinaris
EDITREMOVE
Holds app data for the container. Not for temp plotting or final plot storage.
plots:
/mnt/disk1/
EDITREMOVE
Holds completed Chia plots for farming.
plotting:
/mnt/cache/
EDITREMOVE
Your fast temp space for plotting, ideally a cache pool or unassigned SSD.
chia_protocol:
8444
EDITREMOVE
Sync port to the network. Port-forward on your router!
web:
8926
EDITREMOVE
Machinaris local web app port.
plots_dir:
192.168.1.213_Public
EDITREMOVE
A colon-separated list of all in-container folders holding your completed plots.
flax_protocol:
6888
EDITREMOVE
Sync port to the network. Port-forward on your router!
 
 Show more settings ...
 
 Show docker allocations ...
 
 Add another Path, Port, Variable, Label or Device
```
# paste your Docker run or Docker compose here
I am unsure of how to do this, since it is run as a container
```

</details>

**Additional context & screenshots**
![machinaris_missing_log_txt](https://user-images.githubusercontent.com/36082805/131890338-ddcff957-0224-4276-85b4-66c3382a1231.jpg)

**Comment by guydavis** [comment]
Thanks for the details report.  Yes, I've seen this and it's annoying for sure.  Unfortunately, the issue seems to be in the Unraid Docker Logs window as [other containers](https://www.reddit.com/r/unRAID/comments/jsx53k/why_is_my_docker_container_log_cut_off_on_the_left/) have reported it as well.  The workaround seems to be just executing:

```
docker logs machinaris
```
from an Unraid console/shell as the full log output is then shown, not cut-off:

![image](https://user-images.githubusercontent.com/12936/132099385-6827a1e3-24d2-4a9a-8c75-09899ad52e9c.png)

**Comment by guydavis** [comment]
Hi!  Thanks again for reporting this.  Turned out that it was related to the use of the `-t` Extra Parameter on the Machinaris docker config in Unraid.  Up until the just released v0.5.5 of Machinaris, Plotman required a tty to function, however I've now corrected that  as part of [a fix for TrueNAS users](https://github.com/guydavis/machinaris/issues/94).  

So, now the `-t` is no longer required  in the "Extra Parameters" field of the Unraid Docker configuration for Machinaris.  A great side-effect of this is that the weird log window truncation that you found above, no longer occurs when this `-t` flag is removed.  Please update to the latest Machinaris and remove the `-t` flag, then Apply in Unraid admin UI.  This should fix it.  

Existing Unraid users will need to update their configs, but I've updated the Unraid application template to skip this flag for new users.  Thanks again!
